incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -22,9 +22,9 @@ env:
   DASH_URL: ""
   # url for the appdaemon dashboard, an alternative interface for home assistant
   TZ: "America/Chicago"
-  LATTITUDE: "94"
-  LONGITUDE: "94"
-  ELEVATION: "1217"
+  LATTITUDE: 46
```

Comment:
It is supposed to have two T ? 
or is it LATITUDE

Reply:
Thanks, that's what I get for trying to whip these changes out on a lunch break.